src/core.ts

```diff
@@ -377,7 +362,7 @@ export class Template {
                 (strings[i] === '') ? document.createComment('') :
                                       document.createTextNode(strings[i]),
                 node);
-            this.parts.push(new TemplatePart('node', index++));
+            this.parts.push({type: 'node', index: index++});
```

Comment:
What does removing the `TemplatePart` class do to code size?

Reply:
I don't remember the delta on this specific part, but it was smaller by a few bytes. I like not using a class for this internal storage too. Object literals are minutely faster.